scripts/test-runner.sh

```diff
@@ -21,6 +21,19 @@ function keep_alive() {
     done
 }
 
+# Compare test name with glob expression using extended glob patterns
+# $1 - test name
+# $2 - glob pattern
+function test_name_relevancy() {
+    # See: http://wiki.bash-hackers.org/syntax/pattern#extended_pattern_language
+    shopt -s extglob
```

Comment:
if we want to preserve the environment, it should first query the state of the setting and then set it, and then restore it, otherwise it will disable it if it is run on a system with it enabled